platform-enterprise_versioned_docs/version-25.1/enterprise/studios.mdx

```diff
@@ -0,0 +1,251 @@
+
+The latest environment versions are listed below:
+
+- JupyterLab: `public.cr.seqera.io/platform/data-studio-jupyter:4.2.5-0.8`
+- RStudio Server: `public.cr.seqera.io/platform/data-studio-rstudio:4.4.1-0.8`
+- Visual Studio Code: `public.cr.seqera.io/platform/data-studio-vscode:1.93.0-0.8`
+- Xpra: `public.cr.seqera.io/platform/data-studio-xpra:6.2-r2-1-0.8`
+
+When adding a new Studio, the latest environment versions are tagged `recommended`, and earlier compatible versions are tagged `deprecated`.
+
+Security scans and container inspection reports (including container specifications, configuration, and manifest) are available on-demand at public.cr.seqera.io/platform for each environment images by selecting the `Scan` and `Inspect` icons respectively.
+
+## Docker Compose
+
+This guide assumes that all services will be run in the same container as the rest of your Seqera Platform services.
+
+### Prerequisites
+
+- Allow inbound traffic to port 9090 on the EC2 instance
+- Allow traffic on port 9090 through the AWS LB (Load Balancer)
+- An AWS Route53 wildcard DNS record, such as `*.<seqera_platform_domain>`
+
+### Procedure
+
+1. Download the Studios [environment configuration file](./_templates/docker/data-studios.env).
+
+1. Create an initial OIDC registration token, which can be any secure random string. For example, using openssl:
+
+    ```
+    oidc_registration_token=$(openssl rand -base64 32 | tr -d /=+ | cut -c -32)
+    ```
+
+1. Generate an RSA public/private key pair. A key size of at least 2048 bits is recommended. For example, use `openssl` to generate the key pair:
+
+    ```shell
+    openssl genrsa -out private.pem 2048
+    openssl rsa -pubout -in private.pem -out public.pem
+    ```
+
+1. Download the [data-studios-rsa.pem](./_templates/docker/data-studios-rsa.pem) file and replace its contents with the content of your private and public key files, in the same order (private key on top, public key directly beneath it). Save the file as `data-studios-rsa.pem`, in the same directory as your `docker-compose.yml` file. 
+
+1. Open the `docker-compose.yml` and uncomment the volume mount for the PEM key file for the `backend` service in the `volumes` list. Your PEM file must be named `data-studios-rsa.pem`.
+
+    ```yaml
+    volumes:
+      - $PWD/tower.yml:/tower.yml
+      # An RSA key is required for Studios functionality. Uncomment the line below to mount the key.
+      #- $PWD/data-studios-rsa.pem:/data-studios-rsa.pem
+    ```
+
+1. Open `data-studios.env` in an editor, and make the following changes:
+
+    1. Uncomment the `connect-proxy` and `connect-server` services.
+    1. Set the following environment variables:
+        - `PLATFORM_URL`: The same value assigned to `TOWER_SERVER_URL`. For example, `https://example.com`.
```

Comment:
Also non-blocking: using Platform in newer environment variable names while we still have tower environment variables everywhere else is a confusing UX. We should change the rest to Platform soon, though I know that's a bigger can of worms than it sounds.